cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -42,6 +42,7 @@ const (
 	migrateFlagFormat          = "format"
 	migrateFlagLog             = "log"
 	migrateFlagRevisionsSchema = "revisions-schema"
+	migrateFlagCreateRevisions = "create-revisions"
```

Comment:
Don't you think `create-revisions` is not that correct for this flag? Revision is a migration revision but this flag indicates we create the schema and table for storing the revisions. 

cc @rotemtam and @hilakashai for help.